src/PreviewOkr/components/PreviewModal.tsx

```diff
@@ -0,0 +1,120 @@
+import Modal from '@components/Modal';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import useModal from '@hooks/useModal';
+import { useState } from 'react';
+
+const PreviewModal = () => {
+  const [addOkrOption] = useState<'직접 생성하기' | '가이드에 따라 설정하기'>(
+    '가이드에 따라 설정하기',
+  );
+  const { modalRef, handleShowModal } = useModal();
+
+  return (
+    <>
+      <button style={{ color: 'red' }} type="button" onClick={handleShowModal}>
+        CLICK ME!!!
+      </button>
+      <Modal ref={modalRef}>
+        <div css={modalForm}>
+          <AddOkrOptionText>{addOkrOption}</AddOkrOptionText>
+          <StMainTextContainer>
+            <p>목표 저장 전 내용을 수정하고</p>
+            <p>KR을 달성할 수 있는 task를 추가해보세요!</p>
+          </StMainTextContainer>
+          <StSubTextContainer>
+            <p>KR에서는 수치값을 제외한 텍스트만 수정이 가능하며,</p>
+            <p>task는 KR 당 최대 3개까지 추가 가능합니다.</p>
+          </StSubTextContainer>
+          <form method="dialog">
+            <StConfirmButton>확인</StConfirmButton>
+          </form>
+          <ProgressBox>
+            {addOkrOption === '직접 생성하기' && <ProgressLeftText>5/5</ProgressLeftText>}
+            {addOkrOption === '가이드에 따라 설정하기' && <ProgressLeftText>7/7</ProgressLeftText>}
```

Comment:
```suggestion
const ProgressStep = addOkrOption === '직접 생성하기' ? '5/5' : '7/7'
//여기서 선언해서 사용
// 혹은 명확하게 하려면 switch case

           <ProgressLeftText>{ProgressStep}<ProgressLeftText>
     
```
이렇게 사용해도 좋을것 같네요!

근데 이 부분 제가 로직 연결하면서 어차피 수정해야 할것 같으니 안고쳐도 좋을것 같습니다 감사합니다